openid-caep-1_0.md

```diff
@@ -822,6 +822,61 @@ The following is a non-normative example of a Session Presented event:
     }}
 ~~~
 
+## Risk Level Change {#risk-level-change}
+Event Type URI:
+
+`https://schemas.openid.net/secevent/caep/event-type/risk-level-change`
+
+A vendor may deploy mechanisms to gather and analyze various signals associated with subjects such as users, devices, etc. These signals, which can originate from diverse channels and methods beyond the scope of this event description, are processed to derive an abstracted risk level representing the subject's current threat status.
+
+The Risk Level Change event is employed  by the Trasnmitter to communicate any modifications in a subject's assessed risk level at the time indicated by the `event_timestamp` field in the Risk Level Change event. The Transmitter may generate this event to indicate:
+
+* User's risk has changed due to potential suspecious access from unknown destination, password compromise, addition of strong authenticator or other reasons.
+* Device's risk has changed due to installation of unapproved software, connection to insecure pheripheral device, encryption of data or other reasons.
+* Any other subject's risk changes due to variety of reasons.
+
+
+### Event Specific Claims {#risk-level-change-event-specific-claims}
+The following optional claims MAY be present in a Session Presented event:
+
```

Comment:
Could we add a field that describes the type of risk? You mention in your example user risk and device risk. What if we had a field called `risk_type` whose values were an enum:
- DEVICE
- USER
- ACCOUNT
- OTHER

There may be other values that we should include in this enum as well.

Reply:
@FragLegs I thought we could drive the type using the subject claim. With a single enum it would be hard to flag a single risk event if it applies to more than 1 entity like USER + DEVICE etc.

Reply:
@tulshi I described that the existing subject of any CAEP event describes who this event is for, and it is clarified in [this](https://openid.github.io/sharedsignals/openid-sharedsignals-framework-1_0.html#name-subject-members) section. In other words, if the risk level change event has 2 subjects, a user and a device, then it should be perceived as risk being associated with both entities.

Reply:
I guess my question is let's say there is a user X, who has a phone number Y. The transmitter wants to send out a risk level change about the user (and not their phone), but wants to describe the user using their phone number (i.e. the subject format is the ["phone_number"](https://www.rfc-editor.org/rfc/rfc9493.html#name-phone-number-identifier-for)), then how can the Transmitter convey this information?

Reply:
Thanks, @tulshi, for your comment.

I think this could also be a similar situation for session-related events.

e.g. There are separate sessions created with the user as a principal and the device as a principal.

> The transmitter wants to send out a session revoked/presented/established about the user (and not their device), but wants to describe the user using their phone number (i.e. the subject format is the ["phone_number"](https://www.rfc-editor.org/rfc/rfc9493.html#name-phone-number-identifier-for))

Reply:
A `principal` claim works for me!